DS-Session4-Question1/src/meta/utility/NaturalSort.java

```diff
@@ -0,0 +1,29 @@
+package meta.utility;
+
+
+public class NaturalSort implements Comparable<NaturalSort> {
```

Comment:
I think, this class shall contain only compare method.
Take reference from: https://www.geeksforgeeks.org/collections-sort-java-examples/